utkdigitalinitiatives/islandora_scholar
# address TRAC-829: or... attempt to address 829. Add a the 'thesis_man…

…ager' to the $users array.

**JIRA Ticket**: [TRAC-829](https://jira.lib.utk.edu/browse/TRAC-829)

# What does this Pull Request do?
Adds the thesis_manager to the $users array. Caveat: I'm not sure if this is the best way to do this -- feedback would be appreciated.

# How should this be tested?
* `git pull --all` and switch branches to TRAC-829
* synchronize the repository with your VM
* as userA, submit an ETD
* as thesis_manager,  accept and publish the ETD, then embargo using the object-level embargo
* make sure that you can view the embargo'd ETD as thesis_manager, but not as an anonymous user (or userB, or whatever)

# Interested parties
@robert-patrick-waltz
